charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -344,17 +344,31 @@ processors:
     - sources:
       - from: connection
     extract:
+      otel_annotations: true
       metadata:
-        - "k8s.namespace.name"
-        - "k8s.deployment.name"
-        - "k8s.statefulset.name"
-        - "k8s.daemonset.name"
-        - "k8s.cronjob.name"
-        - "k8s.job.name"
-        - "k8s.node.name"
-        - "k8s.pod.name"
-        - "k8s.pod.uid"
-        - "k8s.pod.start_time"
+        - k8s.namespace.name
```

Comment:
FYI I changed the order of the attributes to strictly align with the preset of the otel-kube-stack, just copy/pasting the code.